common/wireaddr.h

```diff
@@ -22,18 +23,23 @@
  *             where `checksum = sha3(".onion checksum" | pubkey || version)[:2]`
  */
 
+#define	TOR_V2_ADDRLEN 12
+#define	TOR_V3_ADDRLEN 37
+#define	FQDN_ADDRLEN 255
+
 enum wire_addr_type {
 	ADDR_TYPE_PADDING = 0,
 	ADDR_TYPE_IPV4 = 1,
 	ADDR_TYPE_IPV6 = 2,
-};
+	ADDR_TYPE_TOR_V2 = 3,
+	ADDR_TYPE_TOR_V3 = 4
+	};
```

Comment:
According to the original line which is not indented, and 459 other similar lines I would suggest keeping the closing line of enum bare (without indentation).

```
~/src/lightning$ git grep '^};` | wc -l
460
~/src/lightning$ git grep '.+};' | wc -l
0
```